deploy.sh

```diff
@@ -34,7 +34,7 @@ GRAFANA_DOCKER_IMAGE="grafana/grafana:latest"
 KRUIZE_DOCKER_REPO="dinogun/kruize"
 KRUIZE_DOCKER_IMAGE=${KRUIZE_DOCKER_REPO}:${KRUIZE_VERSION}
 
-MINIKUBE_MANIFEST="manifests/minikube/kruize-service-monitor.yaml"
+SERVICE_MONITOR_MANIFEST="manifests/serviceMonitor/kruize-service-monitor.yaml"
```

Comment:
No camel casing for script names/dirs please. `servicemonitor` should be fine.

Reply:
Changed dir name.